### ISSUE TITLE
Broken console output configuration via gradle.properties

### ISSUE THREAD

**Issue by nerro** [gradle/gradle#3053]
I want to use `console` plain in all my gradle projects. So I added `org.gradle.console=plain` to `~/.gradle/gradle.properties`. But I have rich output instead of. Setting this property in project directory (gradle.properties) doesn't work too, only command-line parameter `--console=plain` works.

### Expected Behavior
Console output depending on `org.gradle.console` property defined in `gradle.properties` file.

### Current Behavior
Console output `rich` is used.

### Steps to Reproduce (for bugs)
1. Create a simple project with `gradlew wrapper --gradle-version=4.2`
2. Add `org.gradle.console=plain` to your `~/.gradle/gradle.properties` file
3. Run `gradlew clean` (you can see rich output)
4. Run `gradlew clean --console=plain` (you can see plain output)

### Your Environment

`gradlew --version` output

```bash
------------------------------------------------------------
Gradle 4.2
------------------------------------------------------------

Build time:   2017-09-20 14:48:23 UTC
Revision:     5ba503cc17748671c83ce35d7da1cffd6e24dfbd

Groovy:       2.4.11
Ant:          Apache Ant(TM) version 1.9.6 compiled on June 29 2015
JVM:          1.8.0_121 (Oracle Corporation 25.121-b13)
OS:           Windows 7 6.1 amd64
```

**Comment by oehme** [comment]
Duplicate of https://github.com/gradle/gradle/issues/2744, this is only available in the upcoming Gradle 4.3

**Comment by nerro** [comment]
oh, my bad. thx for info.
